packages/common-db/src/lib/common/tools.ts

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// we currently have no typings so ignore
```

Comment:
See above. Either use `@ts-expect-error` to suppress this while we do not have typings.
Or add a dummy type declaration (but that can be forgotten) in this package:
```
// in a .d.ts file in src:
declare module "@iobroker/js-controller-common" {}
```